ntc_templates/templates/cisco_xr_show_bgp_vrf_all_neighbors_advertised-routes.textfsm

```diff
@@ -0,0 +1,24 @@
+Value NETWORK (\d+\.\d+\.\d+\.\d+)
+Value PREFIX_LENGTH (\d+)
+Value NEXT_HOP (\d+\.\d+\.\d+\.\d+)
+Value FROM ((\d+\.\d+\.\d+\.\d+)|\S+)
+Value AS_PATH (.*)
```

Comment:
@jorlandobr
I'm re-raising this.
Thoughts?

The AS Path will have at least one number so we wouldn't want to accept zero characters, right?

```suggestion
Value AS_PATH (.+)
```